scripts/clean.sh

```
@@ -118,9 +118,11 @@ clean_runtime() {
            target/debug/boxlite-shim
     if [ -z "${KEEP_GUEST_BIN:-}" ]; then
         rm -rf target/$GUEST_TARGET/release/boxlite-guest \
-               target/$GUEST_TARGET/debug/boxlite-guest
+               target/$GUEST_TARGET/release/guest-rootfs \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)